{"name": "test_huggingface_model_loading", "status": "passed", "description": "Test HuggingFace model loading error.", "attachments": [{"name": "log", "source": "8dd3fdf7-2330-4c67-8fe4-60f240243c65-attachment.txt", "type": "text/plain"}], "start": 1757656824019, "stop": 1757656824019, "uuid": "4afe0de9-1a69-4c2c-b035-77e614d72c8e", "historyId": "c057c168d18753251139580dae16491b", "testCaseId": "c057c168d18753251139580dae16491b", "fullName": "tests.unit.test_standardized_llm_errors.TestStandardizedErrors#test_huggingface_model_loading", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "parentSuite", "value": "tests.unit"}, {"name": "suite", "value": "test_standardized_llm_errors"}, {"name": "subSuite", "value": "TestStandardizedErrors"}, {"name": "host", "value": "Joshuas-MacBook-Pro-2.local"}, {"name": "thread", "value": "37492-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.unit.test_standardized_llm_errors"}], "titlePath": ["tests", "unit", "test_standardized_llm_errors.py", "TestStandardizedErrors"]}